src/main/routes/home.ts

```diff
@@ -1,10 +1,7 @@
 import * as express from 'express';
+import { Paths } from '../app/paths';
 
-const router = express.Router();
-
-/* GET home page. */
-router.get('/', (req, res) => {
-  res.render('home');
-});
-
-module.exports = router;
+export default express.Router()
+  .get(Paths.homePage.uri, (req, res) =>{
+    res.redirect(Paths.landing.uri);
+  });
```

Comment:
Surely it's possible to have the logic from `landing.ts` act as a middleware method on the route itself rather than invoke it via a redirect?

Reply:
it's possible but this is cleaner. if moved here there will be logic behind root path ('/') which is not obvious.

Reply:
I'd disagree. I think having middleware such as `app.get('/', checkIsLoggedIn);` is much cleaner than validating via a separate out redirect. It would also mean we do not need to hardcode a success route as we would just call `next()` if the user is signed in. Also has the benefit of being reusable on other routes if we need it.

Reply:
Not a blocker though this approach also makes sense.

Reply:
Not sure what to say here, so we agree to disagree? :)

Reply:
I'd really like to see this changed. I think middleware is the approach we should use here.

Reply:
not sure if we are on same page, it is route level middleware with 1 extra redirect in between. Run app locally, see what happens if you go to root page. I already offered to get on zoom to talk about this, we can do this anytime.

Reply:
doesn't seem like a big change to refactor if i can understand what the problem really is!

Reply:
I think that redirect itself here is fine as / can lead to /landing. But  James is right, auth check should be done in generic way e.g.

express.Router().get("/*" , checkAuth())

and not only on landing page

if user access e.g. /case it should enforce authentication and now it let user in

Reply:
ofcourse not only on landing page - 

check index.ts file - ```enableFor (app: express.Express): void {
    app.all('/case/*', this.requestHandler());
  }
```

Reply:
we don't have any route files in case at the moment but every route file will be checked for auth token.. That's the idea anyway..

Reply:
i guess it will make more sense to me and can explain better when i have a page to go to, since we now have a task-list page in master.. let me use that to showcase .. it will just take some extra time.

Reply:
I wish gulp was still working to show you guys in debug mode. We have changed to webpack now, so will have to work on setting up nodemon and make debug work when we get sometime. For now, i have added some console.logs. Run the app, go to incognito and try to go to https://localhost:3000/case/task-list directly. 

Then in normal chrome, go to home page, login , it will take you to task-list. 

Will clean up all this once you guys are happy with the way it works, if not we need some discussion and some refactoring.

Reply:
@KiranVarma02 I caught up with @tomaszpowroznik. Let's leave this for now. If we find we need to refactor in the future we can do it then.